sql/builtin_functions.cc

```diff
@@ -3329,7 +3329,7 @@ bind(const std::vector<BoundSqlExpression> & args,
                 }
             };
 
-        result.resultInfo = std::move(bound.info);
+        result.resultInfo = bound.info->getConst(isConstant && bound.info->isConst());
```

Comment:
Why do we need to do set constness after the creation of the BoundSqlExpression?  Why can this call
```
 BoundSqlExpression bound = parsed->bind(evalScope);
```
return the info with the constness already set?